exaerror-1.0.1-1.rockspec

```diff
@@ -1,9 +1,9 @@
 package = "exaerror"
-version = "1.0.0-1"
+version = "1.0.1-1"
```

Comment:
`-1` is inteded?

Reply:
Yes, that is the revision number of the spec and unfortunately mandatory.